examples/window-app/common/window-app.matter

```diff
@@ -2231,37 +2231,37 @@ endpoint 2 {
     callback attribute generatedCommandList;
     callback attribute acceptedCommandList;
     callback attribute attributeList;
-    ram      attribute featureMap;
+    ram      attribute featureMap default = 0;
     callback attribute clusterRevision default = 1;
   }
 
   server cluster WindowCovering {
     ram      attribute type default = 0x08;
     ram      attribute physicalClosedLimitLift default = 0xFFFF;
     ram      attribute physicalClosedLimitTilt default = 0xFFFF;
-    persist  attribute currentPositionLift;
-    persist  attribute currentPositionTilt;
-    persist  attribute numberOfActuationsLift;
-    persist  attribute numberOfActuationsTilt;
+    persist  attribute currentPositionLift default = 0;
+    persist  attribute currentPositionTilt default = 0;
+    persist  attribute numberOfActuationsLift default = 0x0000;
+    persist  attribute numberOfActuationsTilt default = 0x0000;
     persist  attribute configStatus default = 0x03;
-    persist  attribute currentPositionLiftPercentage;
-    persist  attribute currentPositionTiltPercentage;
-    ram      attribute operationalStatus;
+    persist  attribute currentPositionLiftPercentage default = 0;
+    persist  attribute currentPositionTiltPercentage default = 0;
+    ram      attribute operationalStatus default = 0x00;
     ram      attribute targetPositionLiftPercent100ths;
     ram      attribute targetPositionTiltPercent100ths;
-    ram      attribute endProductType;
-    persist  attribute currentPositionLiftPercent100ths;
-    persist  attribute currentPositionTiltPercent100ths;
-    persist  attribute installedOpenLimitLift;
+    ram      attribute endProductType default = 0x00;
+    persist  attribute currentPositionLiftPercent100ths default = 0;
+    persist  attribute currentPositionTiltPercent100ths default = 0;
+    persist  attribute installedOpenLimitLift default = 0x0000;
     persist  attribute installedClosedLimitLift default = 0xFFFF;
-    persist  attribute installedOpenLimitTilt;
+    persist  attribute installedOpenLimitTilt default = 0x0000;
     persist  attribute installedClosedLimitTilt default = 0xFFFF;
-    persist  attribute mode;
-    ram      attribute safetyStatus;
+    persist  attribute mode default = 0x0;
+    ram      attribute safetyStatus default = 0x0000;
     callback attribute generatedCommandList;
     callback attribute acceptedCommandList;
     callback attribute attributeList;
-    ram      attribute featureMap default = 0x0017;
+    ram      attribute featureMap default = 0x001F;
```

Comment:
Featuremap changed default value from old constant.

Looking at the zap file, FeatureMap attribute shows up as both 0x0017 and 0x001F for 'server' side, difference being in location: 
  - one at `"name": "MA-windowcovering",` at line 6238 and one at line 7774 which seem to correspond to different endpoints.
  
  New code looks correct in this case, as different endpoints want different defaults.